python/adjoint/objective.py

```diff
@@ -132,4 +133,216 @@ def get_evaluation(self):
         try:
             return self.eval
         except AttributeError:
-            raise RuntimeError("You must first run a forward simulation before resquesting an eigenmode coefficient.")
\ No newline at end of file
+            raise RuntimeError("You must first run a forward simulation before resquesting an eigenmode coefficient.")
+
+
+
+class Fourier_Coefficients(ObjectiveQuantitiy):
```

Comment:
Maybe call it `FourierCoefficients` (omit the underscore) to match the rest of the codebase.

Reply:
Maybe FourierFields and Near2FarFields